section3/exercises/loops-arrays.js

```diff
@@ -18,22 +18,35 @@ for (var i = 0; i < animals.length; i++) {
 
 // YOU DO: Write code below that iterates through a list of animals and prints "The
 // <animal> is awesome!" for each animal.
+var animals = ['Zebra', 'Giraffe', 'Elephant']
 
+for (var i = 0; i < animals.length; i++) {
+  console.log('The ' + animals[i] + ' is amazing')
+}
 
 
 // YOU DO: Create an array of foods and then iterate over that array to log "Add
 // <food> to shopping list" for each food item.
+var food = ['tofu', 'broccoli', 'rice', 'soy sauce', 'peppers']
 
+for (var i = 0; i < food.length; i++){
+  console.log('Add ' + food[i] + ' to shopping list')
+}
 
 
 // YOU DO: Create an array of numbers and then iterate over that array to log double
 // of each of the number. (Ex: 4 doubled is 8.)
-
+var numbers= [1,2,3,4,5,6,7,8]
+for (var i = 1; i <= 8; i++) {
+  console.log(numbers.length *= 2)
```

Comment:
Yeah this one was really hard for me. I played around with it in my terminal for an hour and researched on google. I was too embarrassed to ask my cohort mates in slack because I was working on this exercise the day the project was due. After I turned it in, I did ask a classmate that I made friends with and she explained it and it was an aha! moment for sure. The ONLY thing I hadn't tried was putting numbers.length in the for statement because I didn't know or think to do/try that.

Reply:
Love it! Great use of your resources